### PR TITLE
Add 1.x support docs to README

### LINKED ISSUE #1916: Document long-term support plan and workflow for v1.x
Before we release 2.0 we need to clearly communicate our policy for supporting the 1.x release branch.

### TODO
- [ ] Research how other, similar projects do this
- [ ] Document how we plan to do it in `RELEASE.md`
- [ ] Write it up as an update for the site

### PR DESCRIPTION
**Pull request by maya**
TODO: Add a nav item to the side navigation in the `uswds-site` repo.

Fixes #1916.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,6 +388,10 @@ npm run watch
 Many of our Fractal view templates are compatible with [Nunjucks](https://mozilla.github.io/nunjucks/) (for JavaScript/Node), [Jinja](http://jinja.pocoo.org/docs/2.9/) (Python), and [Twig](https://twig.sensiolabs.org/) (PHP) out of the box. Components that reference other components use a Fractal-specific `{% render %}` tag that will either need to be implemented in other environments or replaced with the appropriate `{% include %}` tags.
 
 
+## Longtime support of v1.x
+Version 1.x is in [maintenence mode](https://en.wikipedia.org/wiki/Maintenance_mode) and is supporting critical bug fixes for 9 months (through January 2020).
```
Comment (thisisdano) [inline README.md]: - Maybe just say `through January 2020` and leave out the 9 months bit?
- Use `will be` instead of `is` I think

Comment (maya) [reply README.md]: Done in 82e1f55.

File: README.md [hunk 2]
```diff
@@ -388,6 +388,10 @@ npm run watch
 Many of our Fractal view templates are compatible with [Nunjucks](https://mozilla.github.io/nunjucks/) (for JavaScript/Node), [Jinja](http://jinja.pocoo.org/docs/2.9/) (Python), and [Twig](https://twig.sensiolabs.org/) (PHP) out of the box. Components that reference other components use a Fractal-specific `{% render %}` tag that will either need to be implemented in other environments or replaced with the appropriate `{% include %}` tags.
 
 
+## Longtime support of v1.x
```
Comment (thisisdano) [inline README.md]: One more thing: Should we say `Long-term support` per https://en.wikipedia.org/wiki/Long-term_support?

Comment (maya) [reply README.md]: Fixed in 6593dd0

**Pull request merged**